test/e2e/common/container_probe.go

```diff
@@ -84,6 +84,46 @@ var _ = framework.KubeDescribe("Probing container", func() {
 		Expect(restartCount == 0).To(BeTrue(), "pod should have a restart count of 0 but got %v", restartCount)
 	})
 
+	/*
+		Testname: Pod readiness probe, with tty set to true for probe exec commands
+		Description: Create a test with a readiness probe exec command that uses "/bin/sh -i -c" and ensure that an ioctl error does't occur and the pod runs successfully
+	*/
+	framework.ConformanceIt("with readiness probe tty set to true for exec commands that use /bin/bash -i -c [NodeConformance]", func() {
```

Comment:
Can't add a new conformance test from scratch, must add this as a regular e2e test first